shared/api/storage_pool_volume_snapshot.go

```diff
@@ -0,0 +1,24 @@
+package api
+
+// StorageVolumeSnapshotsPost represents the fields available for a new LXD storage volume snapshot
+//
+// API extension: storage_api_volume_snapshots
+type StorageVolumeSnapshotsPost struct {
+	Name string `json:"name" yaml:"name"`
+}
+
+// StorageVolumeSnapshotPost represents the fields required to rename/move a LXD storage volume snapshot
+//
+// API extension: storage_api_volume_snapshots
+type StorageVolumeSnapshotPost struct {
+	Name string `json:"name" yaml:"name"`
+}
+
+// StorageVolumeSnapshot represents a LXD storage volume snapshot
+//
+// API extension: storage_api_volume_snapshots
+type StorageVolumeSnapshot struct {
+	Name        string            `json:"name" yaml:"name"`
+	Config      map[string]string `json:"config" yaml:"config"`
+	Description string            `json:"description" yaml:"description"`
```

Comment:
We're missing a way to update this field it looks like.

For this we'd need:
 - UpdateStorageVolumeSnapshot
 - PUT /1.0/storage-pools/POOL/TYPE/VOLUME/snapshots/NAME handler
 - StorageVolumeSnapshotPut struct (with just Description in it)

Reply:
Added. :)